core/interfaces/FactoryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace EventEspresso\core\interfaces;
+
+/**
+ * Interface FactoryInterface
+ *
+ * @package EventEspresso\core\interfaces
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+interface FactoryInterface
+{
+    /**
+     * @param string $fqcn  Fully Qualified Class Name
+     * @return object       Instantiated object
+     * @since $VID:$
+     */
+    public static function create($fqcn);
+}
```

Comment:
We already have `EventEspresso\core\domain\services\factories\FactoryInterface`  I see they differ slightly, but that seems like it could cause confusion?

Reply:
oh.. hmmm... think I missed that... 
ya I agree that it could be confusing, so I'll get rid of `core/interfaces/FactoryInterface` in favour of `EventEspresso\core\domain\services\factories\FactoryInterface`